interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/introduction-to-banded-reports.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Introduction to Banded Reports
+owner: Mary Sammal
+---
+# Introduction to Banded Reports
+
+Banded reports provide a generalized report layout notion. A report document is generated based on the report layout and data source.
+
+## Report Bands
+
+A report layout consists of bands that contain report controls and define their location on document pages. A blank report contains the following bands:
+
+- The **Detail Band** displays recurring contents from the report's data source. This band is printed as many times as there are records available in a data source unless you filtered the data.
+
+    Every report must have a detail band, and you cannot delete it.
+
+- The top and bottom page **Margin bands**. These bands are repeated once on every document page.
+
+![eurd-win-report-bands-design-time-default-layout](../../../images/eurd-win-report-bands-design-time-default-layout.png)
+
+You can also add the following bands:
+
+- **Report Header** and **Report Footer**  
+
+    The **Report Header** is the report's first band (margins are "out-of-page" zones). Use this band to display the report's name, company logo, [date of creation, username](add-extra-information.md), etc.
+
+    The **Report Footer** finalizes the report's informative part. It is placed before the Page Footer and Bottom Margin on the report's last page. You can use the Report Footer band for report [summaries](shape-report-data\shape-data-expression-bindings\calculate-a-summary.md) or conclusions.
```

Comment:
"informative part" Explain.

Reply:
I realized that a Report Footer does not finalizes the report. The Page Footer finalizes. So, I'll change to 
The **Report Footer** is placed before the Page Footer and Bottom Margin on the report's last page. ...
Ok?

Reply:
Ok